etcdctl/ctlv3/command/ep_command.go

```diff
@@ -89,8 +89,8 @@ func epHealthCommandFunc(cmd *cobra.Command, args []string) {
 			ep := cfg.Endpoints[0]
 			cli, err := v3.New(*cfg)
 			if err != nil {
-				fmt.Printf("%s is unhealthy: failed to connect: %v\n", ep, err)
```

Comment:
seems better to have something like @koorgoo's suggestion to avoid terminating on the first error:
```go
errc := make(chan error, len(cfgs))
for _, cfg := range cfgs {
   wg.Add(1)
   go func(cfg *v3.Config) {
       defer wg.Done()
       cli, err := v3.New(*cfg)
       defer func() { errc <- err }
       ...
   }(cfg)
}
wg.Wait()
for i := 0; i < len(cfgs); i++ {
      if err := <-errc; err != nil {
        ExitWithError(ExitError, err)
      }
}
```